code/modules/mob/living/carbon/human/life.dm

```diff
@@ -281,7 +281,7 @@
 			speech_problem_flag = 1
 			gene.OnMobLife(src)
 
-	radiation = Clamp(radiation,0,100)
+	var/radiation = Clamp(total_radiation,0,100)

```

Comment:
Issue. This swap doesn't actually decrease total_radiation. So once you irradiate someone, they're going to be stuck with that amount of radiation 5ever.